pandas/core/ops.py

```diff
@@ -582,6 +588,150 @@ def _get_op_name(op, special):
 DataFrame.{reverse}
 """
 
+_flex_comp_doc_FRAME = """
+Flexible wrappers to comparison operators (`eq`, `ne`, `le`, `lt`, `ge`, `gt`).
+
+Equivalent to `==`, `=!`, `<=`, `<`, `>=`, `>` with support to choose
```

Comment:
I'm a bit confused... In the previous dictionaries, you define `op`, `desc`... and you also have `name` which is the key. Shouldn't we use them here? So, we explain in each page the documented method and not all them?

Reply:
To be consistent with examples that run through several of these sibling methods, the intro lists all of them. It seems counterintuitive to have an intro that describes **one** method with examples from several methods.